build_library/catalyst.sh

```diff
@@ -243,7 +243,7 @@ write_configs() {
     info "    stage4.spec"
     catalyst_stage4 > "$TEMPDIR/stage4.spec"
     info "Putting a symlink to user patches..."
-    ln -sf '/var/gentoo/repos/local/coreos/user-patches' \
+    ln -sfT '/var/gentoo/repos/local/coreos/user-patches' \
```

Comment:
Thanks for the explanation, now that you told this, I think we have the problem elsewhere, too, when we set up other symlinks for directories. Would be good to check them and fix it there as well^^ I think the symlink I set up for `/etc/coreos` → `/etc/flatcar`  is one of those examples but I bet there are more…

Reply:
True that, but one step at a time. Added an entry to my TODO list. :)

Reply:
I scanned our repos and the only case where we need this is in `init` which is a bug I found earlier but couldn't fix without the new knowlege from today, I'll create a PR.

Reply:
https://github.com/flatcar/init/pull/88

Reply:
Thanks for doing it.